doc/en/reference/reference.rst

```diff
@@ -1963,7 +1960,7 @@ All the command-line flags can be obtained by running ``pytest --help``::
                             Default marker for empty parametersets
       norecursedirs (args): Directory patterns to avoid for recursion
       testpaths (args):     Directories to search for tests when no files or
-                            directories are given in the command line
+                            directories are given on the command line
```

Comment:
hmm, this looks like we might have had some typo fixes in places where its not helpfull

Reply:
What do you mean, is there a problem with those changes?

Reply:
actually i think i missread the diff
we might want to make regendoc a regular check for the branches to open regendoc fixup prs

Reply:
Yeah... in fact, we could make an action that runs anytime a PR is opened or pushed to, runs regendoc, and that pushes a new commit if any `.rst` files have changed.

Reply:
that unfortunately can't be done safely without exposing privileged credentials to the pull request code or using a third party app to do the pushing

Reply:
@asottile would it be safe enough safe, i we would have to trigger an actual workflow action on main with the pr as parameter (similar to the create pr workflow)

that way the pr code would work similar to the workflow for creating a release

another alternative would be to just always have a release pr on standby (instead of manually triggering them) and using them as indicator